plugins/scaffolder-actions/scaffolder-backend-module-utils/README.md

```diff
@@ -1,15 +1,20 @@
 # scaffolder-backend-module-utils actions package
 
+## Intro
+
 Welcome to the roadie `utils` actions for the `scaffolder-backend`.
 
-This contains a collection of actions:
+This contains a collection of actions to use in scaffolder templates:
+
+- [Zip](#zip)
+- [Sleep](#sleep)
+- [De-serialise a file](#de-serialise)
```

Comment:
This will be better written as `deserialize`, same below. We are all americans nowadays